app/javascript/widget/assets/scss/_variables.scss

```diff
@@ -42,7 +42,7 @@ $color-light-gray: #999a9b;
 $color-border: #e0e6ed;
 $color-border-transparent: rgba(224, 230, 237, 0.5);
 $color-border-light: #f0f4f5;
-$color-border-dark: #cad0d4;
+$color-border-dark:#314156;
```

Comment:
@SamalaSumanth0262 This fix doesn't work since we are not using `color-border-dark` in widget. Please check the input in `CustomerSatisfaction` component.
https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/shared/components/CustomerSatisfaction.vue#L25

Reply:
@muhsin-k I have changed the colour of the border it works both for light and dark mode.

Reply:
Hey @SamalaSumanth0262, We are changing the color of our style guide, which would change it everywhere. You need  to make changes in `CustomerSatisfaction` probably by tailwind or CSS.